mysql-test/main/mdev-34413-icp-reverse-order.result

```diff
@@ -84,8 +84,8 @@ a	b	c
 10	10	10
 SELECT * FROM information_schema.SESSION_STATUS WHERE VARIABLE_NAME LIKE '%icp%';
 VARIABLE_NAME	VARIABLE_VALUE
-HANDLER_ICP_ATTEMPTS	2
-HANDLER_ICP_MATCH	2
+HANDLER_ICP_ATTEMPTS	1
```

Comment:
Why this change? The query above doesn't use the Rowid Filter. 
I assume this will be gone once JOIN-READ-LAST-KEY-SHOULD-BE-IN-ICP suggestion is addressed.